src/components/unique-token/UniqueTokenAttributeItem.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { UniqueAssetTrait } from '../../entities/uniqueAssets';
+import Tag from '../Tag';
+import transformUniqueAssetTraitsForPresentation from '@rainbow-me/helpers/transformUniqueAssetTraitsForPresentation';
+import { magicMemo } from '@rainbow-me/utils';
+
+type AttributeItemProps = UniqueAssetTrait &
+  ReturnType<typeof transformUniqueAssetTraitsForPresentation> & {
+    color: string;
+    slug: string;
+    max_value?: string | number;
+  };
+
+const UniqueTokenAttributeItem: React.FC<AttributeItemProps> = ({
+  color,
+  disableMenu,
+  trait_type: type,
+  slug,
+  value,
+  max_value: maxValue,
+  keepLowerCase,
+  originalValue,
+}) => (
```

Comment:
```suggestion
const UniqueTokenAttributeItem = ({
  color,
  disableMenu,
  trait_type: type,
  slug,
  value,
  max_value: maxValue,
  keepLowerCase,
  originalValue,
}) : AttributeProps => (
```

We have a preference for not using `FC`, because this is reducing the way we can type `children`. /cc @markdalgleish

Reply:
Okay! Makes sense I will use just the props type